manual/modules/ROOT/nav.adoc

```diff
@@ -13,6 +13,7 @@
 * xref:plugin-messaging.adoc[New Message Plugin API]
 * xref:gui-model.adoc[Model - Gui code organization]
 * xref:comm-overview.adoc[Communication Overview]
+* xref:coding-guidelines.adoc[Coding Guidelines]
```

Comment:
This is just a draft for now. I can combine "Coding Guidelines" with "Code Formatting" if you prefer.

Reply:
Feel free to make a proposal according to your own ideas. But don't forget the current text on code formatting etc. ,  it needs the wider context you are providing here.

Reply:
I combined coding guidelines and code formatting:

![image](https://github.com/user-attachments/assets/f4955d1b-7cfb-4810-810f-2baaeba829d9)